enhancements/single-node-production-edge-cluster-profile.md

```diff
@@ -271,6 +271,14 @@ show as degraded.
 oc patch -n openshift-ingress-operator ingresscontroller/default --type=merge --patch '{"spec":{"replicas": 1}}'
 ```
 
+#### machine-config-operator
+
+The machine-config-operator includes a check in code that the number
+of control plane nodes is 3. Removing this or changing the minimum is
+required to allow the operator to complete its work, for example to
+enable the realtime kernel on a single node with the appropriate
+performance profile settings.
```

Comment:
There are some other checks that need to be removed / limited in MCO as well:
- https://github.com/openshift/machine-config-operator/blob/66f332e3079f8fb0a8d3c4e8e31e8417d1524de0/pkg/controller/node/node_controller.go#L899 (check for etcd leader)
- https://github.com/openshift/machine-config-operator/blob/66f332e3079f8fb0a8d3c4e8e31e8417d1524de0/pkg/controller/node/node_controller.go#L957 (maxUnavailable override for master pool)